### PR TITLE
Improve domain experience closes #632

### LINKED ISSUE #632: Improve how we set new domain at community page close #613
- [x] Carregar o DNS para o domínio selecionado, quando estiver visualizando sub-domínios (voltar da api os nameserver de dns)
- [x] Adicionar CLIQUE no domínio para carregar sub domínios e Servidores DNS
- [x] Ao remover, aviso(modal) está aparecendo abaixo do botão "Adicionar novo subdomino externo" 
- [x] Ao remover, aviso(modal) continua aberto se clico em outro elemento da página
- [x] Adicionar mensagem de feedback após verificar se alteração foi feita no servidor DNS ****************** TEXTO

### PR DESCRIPTION
**Pull request by igr-santos**
## Issues

- Improve how we set new domain at community page. Closes #632

### CODE REVIEW

File: client/community/notifications/dns.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+export const checkDNSSuccess = () => ({
```
Comment (lpirola) [inline client/community/notifications/dns.js:1]: @igr-santos maybe it's a good idea to move these messages to intl component?

**Review by lpirola — APPROVED** [review]
Great work!

**Comment by gabrielrtakeda** [comment]
@igr-santos, on the **DNS records** section, if the `value` column is too big, the column moves to an another line.. This is the expected behaviour/layout?

screen shot 2017-06-05 at 12 53 15

May use `<table>` here, would be a good a idea.. Or `display: table`, `display: table-cell` styles for example...

**Comment by gabrielrtakeda** [comment]
@igr-santos, I will check [this](https://github.com/nossas/bonde-client/pull/646#issuecomment-306226740) in the #638

**Review by gabrielrtakeda — APPROVED** [review]

File: client/community/notifications/dns.js (hunk 1 above)
Comment (igr-santos) [reply client/community/notifications/dns.js:1]: @lpirola it's done!

**Pull request merged**